deployment/terraform/dataspace/main.tf

```diff
@@ -40,8 +40,10 @@ locals {
   registration_service_dns_label = "${var.prefix}-registration-mvd"
   edc_default_port               = 8181
 
-  dataspace_did_url = "did:web:${azurerm_storage_account.dataspace_did.primary_web_host}"
-  gaiax_did_url     = "did:web:${azurerm_storage_account.gaiax_did.primary_web_host}"
+  dataspace_did_uri = "did:web:${azurerm_storage_account.dataspace_did.primary_web_host}"
+  gaiax_did_uri     = "did:web:${azurerm_storage_account.gaiax_did.primary_web_host}"
+
+  registration_service_url = "http://${azurerm_container_group.registration-service.fqdn}:${local.edc_default_port}"
```

Comment:
Due to recent change of JWS now the /autority endpoints in registration service are running on a different port (8182) and not on the edc default ports.

See change in this MVD PR which also adjusting registration URL as per new port https://github.com/agera-edc/MinimumViableDataspace/pull/178/files#diff-b87792b7f7cb2c655edf3945f48b0a96729a8c73f9de2e42d2715d547a4e5a9e.

JWS change on registration service already merged in main branch so it will be a good idea to use pot 8182 here as done in above mentioned PR.

Reply:
What about waiting for which Pr will get merged first and then resolve the conflicts by applying the changes from #178?
PR #178 is still in review and it may change so it can be difficult to adapt this one with the changes